DARPA-ASKEM/terarium
# Feat: Auto Map V1

Adding mira call to auto map feature

To Test:

1. Grab a dataset, make sure that it is enriched or enrich it.
2. Grab a model make sure this is enriched as well
3. Open either calibration node and click automap

Sample:
.../projects/5/workflows/811fca52-31ad-4fee-bd2c-5db972af5c14

https://github.com/DARPA-ASKEM/terarium/assets/17088680/75f92cb1-b033-48d8-9996-a96420819bdc


Things to note:
This is using pairwise comparison rather than cluster comparison. Ill start a conversation about this should we care to ask MIRA to change this
